code/modules/mob/living/living.dm

```diff
@@ -1057,4 +1057,32 @@
 
 /mob/living/onTransitZ(old_z,new_z)
 	..()
-	update_z(new_z)
\ No newline at end of file
+	update_z(new_z)
+
+/mob/living/MouseDrop(mob/over)
+	. = ..()
```

Comment:
How does this work with inventory panels.

Reply:
it opens too.